### PR TITLE
Feature/flash context

### PR DESCRIPTION
**Pull request by alexanderdean**

### CODE REVIEW

File: schemas/com.snowplowanalytics.snowplow/flash_context/jsonschema/1-0-0 [hunk 1]
```diff
@@ -36,8 +37,17 @@
...
 			"type": "boolean"
+		},
+		"domainSessionIndex": {
+			"type": "number"
+		},
+		"domainUserId": {
+			"type": "string"
+		},
+		"userFingerprint": {
+			"type": "number"
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/flash_context/jsonschema/1-0-0]: Should this be a number? In `atomic.events` we have `user_fingerprint varchar(50) encode runlength,`

Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/flash_context/jsonschema/1-0-0]: /cc ephraimtabackman

**Comment by alexanderdean** [comment]
Hey @ephraimtabackman - can you update the PR? I think userFingerprint should be a string...

**Comment by ephraimtabackman** [comment]
@alexanderdean -- I changed userFingerprint to type string.

**Comment by alexanderdean** [comment]
Cool - thanks!

**Pull request merged**